src/main/resources/static/js/routes.js

```diff
@@ -1,44 +1,56 @@
 angular.module('Mccy.routes', [
-        'ngRoute'
-    ])
+	'ngRoute'
+])
 
-    .config(function ($routeProvider) {
-        $routeProvider
-            .when('/view', {
-                templateUrl: 'views/view-containers.html',
-                controller: 'ViewContainersCtrl'
-            })
-            .when('/new-server', {
-                templateUrl: 'views/create-container.html',
-                controller: 'NewContainerCtrl'
-            })
-            .when('/upload-mod', {
-                templateUrl: 'views/upload-mod.html',
-                controller: 'UploadModCtrl'
-            })
-            .when('/manage-mods', {
-                templateUrl: 'views/manage-mods.html',
-                controller: 'ManageModsCtrl'
-            })
-            .otherwise('/view')
-    })
+.config(function ($routeProvider) {
```

Comment:
Yeah. I wasn't sure about that. I went with 4 space for most things I wasn't sure about. I'll go ahead and change them back.